cmd/start.go

```diff
@@ -59,6 +59,11 @@ var startCmd = &cobra.Command{
 			}
 		}
 
+		crcCheck := !config.GetBool("ignore-crc")
```

Comment:
`crcCheck` should be consistent with config: use `ignoreCRC` or have the `crcCheck` disabled with `--crc-check=false`